src/inc/pedecoder.inl

```diff
@@ -1100,27 +1100,47 @@ inline void PEDecoder::GetNativeILPEKindAndMachine(DWORD* pdwKind, DWORD* pdwMac
         *pdwMachine = pNativeHeader->Machine;
 }
 
-inline CORCOMPILE_DEPENDENCY * PEDecoder::GetNativeDependencies(COUNT_T *pCount) const
+#endif  // FEATURE_PREJIT
+
+inline CORCOMPILE_DEPENDENCY * PEDecoder::GetNativeDependencies(COUNT_T* pCount) const
```

Comment:
I wonder why we cannot use the same mechanism I have used in Module::IsInSameVersionBubble to enumerate the native AssemblyRefs.

Reply:
Because of we need additional data (GUID in this case) associated with the assemblyref.

Reply:
Ah, I've missed this is getting the MVID part.